app/vodic/vodic-responsive.css

```
@@ -0,0 +1,254 @@
+line-height: 24px; 
+padding-bottom: 10px;
+ }
+ .mobile_local_florists_text p:last-child, .mob_own_website_sec_text p:last-child{ padding-bottom: 0px; }
+ 
+
+ .letter_spacing_3px{ letter-spacing: -0.3px !important; }
+.mob_own_website_sec{ padding-top: 47px; padding-bottom: 58px; background: linear-gradient(173.13deg, #FFFFFF -75.8%, #F2F6F9 195.47%) !important; }
+ .mob_own_website_sec_inner{ max-width: 360px;  padding-left: 5px; padding-right: 7px; padding-top: 0px !important; padding-bottom: 0px !important;  }
+
+
+ .mob_own_website_sec_img{ margin-top: 23px !important; max-width: 187px; height: 250px; margin: auto; }
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)